### DISCUSSION TITLE
Cloudwatch: Allow dimensions per metric

### DISCUSSION THREAD

**Discussion by andresmgot** [grafana/grafana#38340] [Data Sources]
Follow up of #33426

**What would you like to be added**: Dimensions specific to a single metric in a namespace for AWS Cloudwatch.

**Why is this needed**:

At the moment, dimensions in Cloudwatch integration are associated to a namespace (e.g. `AWS/EFS`) but there are some cases in which a single metric contains different dimensions (see this [comment](https://github.com/grafana/grafana/pull/33426#discussion_r631147778) for an example). These dimensions are not available for other metrics of the namespace.

This may require changes both in the frontend and backend since the query that retrieves the list of dimensions doesn't contain the selected metric name.

cc/ @sunker

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!
